### ISSUE TITLE
"Remember Me" functionality is broken after disabling 2FA

### ISSUE THREAD

**Issue by ljlapierre** [BookStackApp/BookStack#2985]
**Describe the bug**
When 2FA was first enabled I turned it on, but it seems to have broken the Remember Me functionality. I use Bookstack to take notes on a fairly regular basis from my phone and desktop, and decided I'd sooner have automatic login over 2FA. I disabled 2FA on my account, but Remember Me does not work.

**Steps To Reproduce**
1. Enable 2FA for your user
2. Log in using Remember Me
3. Disable 2FA for your user
4. Log in using Remember Me

**Expected behavior**
My next bookstack session should have me automatically logged in

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): 21.08.5
 - PHP Version: 7.4
 - Hosting Method (Nginx/Apache/Docker): Apache

**Comment by Bolthier** [comment]
I have the same issue after updating without ever enabling 2FA.

I wasn't sure if it was a Firefox bug or an issue with an addon, so I didn't raise an issue. But after testing for several days with different browser profiles it still logs me out after some time.

Feels like between a few hours or a day, not right after closing the site an reopening it. As if I didn't check the "remember me" box.

**Comment by ssddanbrown** [comment]
@Bolthier You could try setting `SESSION_LIFETIME=1` (So sessions only live for 1 minute by default) in your `.env` then let me know if you remain logged in, when using the "remember me" option, after 2 minutes of inactivity on the homepage.

**Comment by ljlapierre** [comment]
I have yet to verify the cookie, but this behavior is consistent with what I'm seeing. I log in from my home machine, work machine, and from my phone quite frequently.

**Comment by Bolthier** [comment]
> Maybe it'd be a case of a manual logout (Via header option click) revoking all existing "remember me" sessions?

Yeah, that would be the ideal solution. Maybe even give the user some info like browser and ip of all active sessions.

**Comment by ljlapierre** [comment]
The features could be staged, start with a "toast all sessions" button for basic functionality that could be expanded upon easily enough

**Comment by wouterVE** [comment]
I'm having the same behavior as [@Bolthier](https://github.com/BookStackApp/BookStack/issues/2985#issuecomment-948943030)  describes. Logging in another device does invalidate my other Bookstack sessions after the `bookstack_session` cookie has expired. In the past (I think until the version of May?) I did not have this issue.

For now I've 'solved' it by increasing the value of `SESSION_LIFETIME` in the` .env` file.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/2985
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
